python/cudf/bindings/copying.pyx

```diff
@@ -234,3 +235,30 @@ def apply_scatter_array(dev_array, maps, out_col=None):
     in_col = columnops.as_column(dev_array)
     return apply_scatter_column(in_col, maps, out_col)
 
+
+def apply_copy_range(out_col, in_col, idx_out_begin, idx_out_end, idx_in_begin):
+    """Call cudf::copy_range
+
+    Parameters
+    ----------
+    out_col : Column
+        column to copy to
+    in_col : Column
+        column to copy from
+    idf_out_begin, idx_out_end : gdf_index_type
+        [start, end) indices in out_col to copy to
+    idx_in_begin : gdf_index_type
+        index in in_col to start copying from
+    """
+    check_gdf_compatibility(out_col)
+    check_gdf_compatibility(in_col)
+    cdef gdf_column *c_out_col = column_view_from_column(out_col)
+    cdef gdf_column *c_in_col = column_view_from_column(in_col)
```

Comment:
You're leaking both of these, you should free them after the `copy_range(...)` call below.